### DISCUSSION TITLE
Alateen Listings

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1482] [Q&A]
For safety reasons we are not directly listing zoom links on our Alateen meeting listing, I am using a button to get people to an Alateen info page where they can contact the Alteen coordinator for zoom links. If I don't add a zoom link in the TSML UL the meeting lits as inactive.

I want to list these to direct to a specific page, mention the city/region but some are online only so I have to list them as not in-person and that is not working for me.  How can I get it to say online with just the city no specific address?

![image](https://github.com/user-attachments/assets/bd044c50-d2a3-49c1-9a41-7c17898afc48)
Thanks
Rose L

**Comment by ginslo** [comment]
Hi @district6wc, here's a possible solution for you to try. 

1. For the address, enter the city name and state in the Address field without a street address. 
2. For the videoconference URL, enter https://zoom.us/.

![Screenshot from 2024-08-05 22-36-18](https://github.com/user-attachments/assets/47aa6076-6d3a-4422-a635-609f6a61d8c7)

![Screenshot from 2024-08-05 22-22-55](https://github.com/user-attachments/assets/49a7ec4c-d2c4-45c6-977b-c2cfaaf2f16b)

Let me know if that helps.

**Comment by joshreisner** [comment]
hi rose, this issue came up before, did this solution stop working? https://github.com/code4recovery/12-step-meeting-list/discussions/1037

linking to an explanation page on your website would be clearer for users i think

**Reply by ginslo** [reply]
Good morning. Thank you @joshreisner.  #1037  is a more elegant solution for sure. I have verified that it does work on my sample site as well. @district6wc please let us know if that is a workable solution for you or if you need something else. Thanks.

**Comment by district6wc** [comment]
I do have the button, that was a request I made, but if I don't use a zoom link in the URL field it seems to come up as in-person. I tried adding the page I created and it still said in-person. Will try this again.

**Reply by ginslo** [reply]
Yes, it does need a conference link. However rather than putting in the Zoom link, you will put in the link to your Alateen page which exactly matches what you added to functions.php.

**Comment by district6wc** [comment]
Hi I have a question on where the function file should be. I am wondering if that is part of my issue here. We changed our theme and my current code for the plugin meeting types and customization is in the old theme folder (business-roy). The new theme Astra has a function file but it is specific for the theme. Do I have to move the code to this file or can I just leave it in the old inactive theme folder.

**Reply by anchovie91471** [reply]
Yes. For the customer code to run, it always needs to be in the active theme folder.

It's ok to just copy the customizations to the bottom of the astra functions file.

**Comment by district6wc** [comment]
Can it be a 2nd file? I tried adding it to the current function file for Astra and it literally crashed the staging site. It does seem to be working on the staging site the way I have it now, in the folder for the old theme

**Reply by anchovie91471** [reply]
The crash is likely being caused by incomplete code or it being pasted in the wrong place. Can you send the code from the old and the new functions.php to anthony.baggett@gmail.com and I can maybe give you some guidance.

**Comment by district6wc** [comment]
Thanks I sent both function files over. I am not sure if this is an issue
but we made a copy of the theme and I am not sure it is actually a child of
the theme from what I have been googling. Could that be a factor?

Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by anchovie91471** [reply]
I sent the updated functions.php. Please let me know if this solves the issue. Back up your original file before replacing with the one I sent.
